src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
One thing still not very clear to me. `add_intercept` being true override `fit_intercept` which is intended. But `add_intercept` being false will have no effect on `fit_intercept`. Also if user uses `array[1, ...]` and linear kernel and specify `fit_intercept=true`  in kernel params, the algorithm will essentially fit `array[1,1,...]`? I thought the behavior in the linear case is such that `fit_intercept` in the kernel params is a `no-op` while in the kernel case `fit_intercept` overrides. There are some inconsistencies due to multiple ways of specifying whether to fit intercept. I am a bit confused and need to think about it more....